pages/resources/faq.md

```diff
@@ -1,38 +1,71 @@
+
+
+## Why develop three clients in parallel?
+
+Having three clients being developed simultaneously in threee languages by independent teams is a great way to bootstrap resilience-by-diversity into the network. If a bug emerges in one client, the network can continue using nodes running the other clients. Having multi-client interoperability as a foundation from day one helps us to avoid building myopically, and also allows for a more diverse range of developers to make open source contributions.
+
+## With Ethereum's state and history constantly growing, won't Portal clients eventually stop being light?
+
+Until statelessness is shipped, Ethereum's state and history will continue to grow, and this will increase the total network storage for Portal Network. However, we anticipate this being ameliorated by growth in adoption of Portal clients that will help distribute that storage more widely. We also anticipate growing our developer community and finding more efficient storage methods. Eventually, Ethereum will ship stateless clients which will solve the issue of state and history growth.
 
-### Full Functionality for Stateless Clients
 
-The core Ethereum protocol is moving towards a "stateless" model of block verification. Under this model a client will be able to fully verify the execution of a block using a witness. Such a client would no longer need to keep or maintain any of the Ethereum "state" data. Such a client is very valuable in the context of the core protocol, as it facilitates a cleaner merge of the Eth1 and Eth2 chains.
-Additional reading on why stateless is so important to the Eth1/Eth2 merge: https://dankradfeist.de/ethereum/2021/02/14/why-stateless.html)
-What is easy to overlook is that such a "stateless" client will be unable to much else without additional infrastructure. Specifically it would be unable to serve the vast majority of the JSON-RPC apis. The Portal Network provides this additional infrastructure, allowing stateless clients to also expose the external APIs that support the web3 ecosystem.
+## How do you know Portal data is from the canonical Ethereum chain?
+
+The canonicality of Portal data is verified using hash accumulators. For pre-merge blocks there are two accumulators: the epoch accumulator and the master accumulator. The epoch accumulator collects the block hashes for every canonical ethereum block in an epoch and constructs a Merkle tree. The root hash for the epoch tree is then added as a leaf to the master accumulator. These structures allow proofs to be generated that specific block hashes are part of the canonical chain. Because The Merge has already happened and the chain of pre-merge blocks is now fixed, the accumulator can be computed once and then stored as a static structure that can be shipped with Portal clients. Anyone can recompute the hash accumulator from Ethereum data.
```

Comment:
`These structures allow proofs to be generated that specific block hashes are part of the canonical chain.` -> `These structures allow proofs to be generated that prove specific block hashes are part of the canonical chain.`

Reply:
```suggestion
The canonicality of Portal data is verified using hash accumulators. For pre-merge blocks there are two accumulators: the epoch accumulator and the master accumulator. The epoch accumulator collects the block hashes for every canonical ethereum block in an epoch and constructs a Merkle tree. The root hash for the epoch tree is then added as a leaf to the master accumulator. These structures allow proofs to be generated that demonstrate that specific block hashes are part of the canonical chain. Because The Merge has already happened and the chain of pre-merge blocks is now fixed, the accumulator can be computed once and then stored as a static structure that can be shipped with Portal clients. Anyone can recompute the hash accumulator from Ethereum data.
```